cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -225,7 +228,7 @@ macro(resolve_dependency DEPENDENCY_NAME)
     list(APPEND FIND_PACKAGE_ARGUMENTS CONFIG)
   endif()
   if(${DEPENDENCY_NAME}_SOURCE STREQUAL "AUTO")
-    find_package(${FIND_PACKAGE_ARGUMENTS})
+    find_package(${FIND_PACKAGE_ARGUMENTS} QUIET)
```

Comment:
Why do you want to add `QUIET` here?
I think that `find_package()` output is useful for debugging, user support and so on.